workers/metadata_clean/worker.go

```diff
@@ -0,0 +1,138 @@
+package metadata_clean
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"github.com/m6yf/bcwork/bcdb"
+	"github.com/m6yf/bcwork/config"
+	"github.com/rotisserie/eris"
+	"github.com/rs/zerolog/log"
+	"github.com/volatiletech/sqlboiler/v4/queries"
+	"os"
+	"strings"
+)
+
+type Worker struct {
+	DatabaseEnv string `json:"dbenv"`
+}
+
+type TransactionIds struct {
+	TransactionId string `json:"transaction_id"`
+}
+
+const fetch_query = `WITH ranked_records AS (
+    select transaction_id, key, updated_at,
+        row_number() over (Partition by key Order by updated_at) AS row_num,
+        count(*) over (partition by key) as total_count
```

Comment:
With this query you will always receive 5 the most old rows for each key no matter how many existed in DB.
What is the purpose of count? If you want to receive everything except last 5 rows `row_number` seems to be enough. Also I think it should be ordered `desc`.
```suggestion
WITH ranked_records AS (
    select transaction_id, key, updated_at,
        row_number() over (Partition by key Order by updated_at desc) AS row_num
    from metadata_queue)
select transaction_id
from
    ranked_records
where
    row_num > 5
ORDER BY
    key,
    updated_at desc;
```

Reply:
this query selects all the rows **except** for the 5 most updated rows
I will test myself

Reply:
5 the most old rows for each key:
![image](https://github.com/user-attachments/assets/b34d4ad9-871e-47cc-90b2-44ef62f9574d)
But, for example, key `badv:1111111` have 2 more to delete:
![image](https://github.com/user-attachments/assets/c9a39577-b453-451a-845e-a0b7fddd3bc1)
For keys with bigger amount of rows a lot of keys remain untouched:
![image](https://github.com/user-attachments/assets/a574bbec-bf5f-4f4a-8da0-22bfe3602a87)